core/src/collision_from_mesh.rs

```diff
@@ -16,27 +18,48 @@ use crate::{CollisionShape, RigidBody};
 ///         .spawn()
 ///         .insert(Transform::default()) // Required to apply GLTF transforms in Bevy
 ///         .insert(GlobalTransform::default())
-///         .insert(PendingConvexCollision {
-///             body_type: RigidBody::Static,
-///             border_radius: None,
-///         })
+///         .insert(PendingConvexCollision::default())
+///         .insert(RigidBody::Static)
+///         .insert(CollisionLayers::default())
 ///         .with_children(|parent| {
 ///             parent.spawn_scene(asset_server.load("cubes.glb#Scene0"));
 ///         });
 /// }
 /// ```
-#[derive(Component, Reflect)]
+#[derive(Component, Clone, Copy, Reflect)]
 pub struct PendingConvexCollision {
     /// Rigid body type which will be assigned to every scene entity.
+    #[deprecated(note = "Insert body type component into the entity with this component")]
     pub body_type: RigidBody,
     /// Border radius that will be used for [`CollisionShape::ConvexHull`].
     pub border_radius: Option<f32>,
 }
 
+#[allow(deprecated)]
+impl Default for PendingConvexCollision {
+    fn default() -> Self {
+        Self {
+            body_type: RigidBody::Static,
+            border_radius: None,
+        }
+    }
+}
+
 /// Generates collision and attaches physics body for all entities with [`PendingConvexCollision`].
+#[allow(deprecated)]
 pub(super) fn pending_collision_system(
     mut commands: Commands<'_, '_>,
-    added_scenes: Query<'_, '_, (Entity, &Children, &PendingConvexCollision)>,
+    added_scenes: Query<
+        '_,
+        '_,
+        (
+            Entity,
+            &Children,
+            &PendingConvexCollision,
+            &RigidBody,
```

Comment:
To preserve behavior, I think we need `Option<&RigidBody>`, so that existing usages continue to work.

Reply:
Oh, right, thanks!